src/did/Web5DID.js

```diff
@@ -15,7 +15,7 @@ class Web5DID {
     return this.#web5;
   }
 
-  async create(method, options = { }) {
+  async create(method = 'key', options = { }) {
```

Comment:
can we go ahead and add JSDoc here? re: https://github.com/TBD54566975/web5-js/pull/32#issuecomment-1500401159